### ISSUE TITLE
0 dependencies reported

### ISSUE THREAD

**Issue by wiyarmir** [dependencyci/support#34]
github.com/wiyarmir/TravisForAndroid reports 0 dependencies but it is not the case

**Comment by andrew** [comment]
Ah you have a different layout in your `build.gradle` file than the one I tested on, I'll see if I can improve my parser to pick up those dependencies.

**Comment by wiyarmir** [comment]
Yes, this layout is quite peculiar for parsing, but quite helpful for development :) Let me know if you need any help from my side.

**Comment by andrew** [comment]
Dependency CI doesn't currently execute any code to parse dependencies, so for gradle I'm using this module to parse the file: https://www.npmjs.com/package/gradle-to-js then this code to load modules from Libraries.io: https://github.com/librariesio/bibliothecary/blob/master/lib/bibliothecary/parsers/maven.rb#L97-L112

I think the npm module will be able to parse it, just need to add a slightly different code path in bibliothecary to handle your setup

**Comment by ctubbsii** [comment]
I'm also seeing this on github.com/ctubbsii/accumulo, a basic multi-module maven project.

**Comment by andrew** [comment]
@ctubbsii I've opened a seperate issue for you here: https://github.com/librariesio/bibliothecary/issues/74

**Comment by andrew** [comment]
@wiyarmir I've opened an issue for improving gradle support on the parsing library here: https://github.com/librariesio/bibliothecary/issues/75

**Comment by wiyarmir** [comment]
Great! Just one catch, if you reference line numbers in the master branch, a future commit might break your link. Try using this link instead in the issue, to that specific commit: https://github.com/wiyarmir/TravisForAndroid/blob/f839343d11d76f002ca057e591f0737aff1bf38d/build.gradle#L73-L131

**Comment by andrew** [comment]
Updated, thanks for the reminder

**Comment by libeclipse** [comment]
This issue also crops up for python repositories. Either less dependencies than the actual number or 0 repositories are reported. Maybe parse requirements.txt and setup.py for dependencies?
